### PR TITLE
[opt](nereids)estimate rowcount for is-null filter when column stats are not available

### PR DESCRIPTION
**Pull request by englefly**
## Proposed changes

Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by englefly** [comment]
run buildall

**Comment by englefly** [comment]
run buildall

**Review by morrySnow — DISMISSED** [review]

**Comment by englefly** [comment]
run buildall

**Review by starocean999 — APPROVED** [review]

File: fe/fe-core/src/main/java/org/apache/doris/nereids/stats/FilterEstimation.java [hunk 1]
```diff
@@ -63,9 +63,8 @@ public class FilterEstimation extends ExpressionVisitor<Statistics, EstimationCo
 
     public static final double DEFAULT_HAVING_COEFFICIENT = 0.01;
 
-    public static final double DEFAULT_EQUALITY_COMPARISON_SELECTIVITY = 0.1;
     public static final double DEFAULT_LIKE_COMPARISON_SELECTIVITY = 0.2;
-    public static final double DEFAULT_ISNULL_SELECTIVITY = 0.001;
+    public static final double DEFAULT_ISNULL_SELECTIVITY = 0.005;
```
Comment (xzj7019) [inline fe/fe-core/src/main/java/org/apache/doris/nereids/stats/FilterEstimation.java:67]: how about not null? Is it necessary to handle, or has been automatically handled?

Comment (englefly) [reply fe/fe-core/src/main/java/org/apache/doris/nereids/stats/FilterEstimation.java:67]: not (A is null) selectivity is : 1 - 0.005

Comment (xzj7019) [reply fe/fe-core/src/main/java/org/apache/doris/nereids/stats/FilterEstimation.java:67]: ok, lgtm

**Review by xzj7019 — APPROVED** [review]

**Pull request merged**